app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -54,6 +74,37 @@ export class GridRow extends Component {
     this.setState({ withAccordion: true });
     this.overflowCell.style.maxHeight = `${this.overflowCellMaxHeight}px`;
   };
+
+  getHighLightBlockHeight = () => {
+    const { rowClassMapper, value } = this.props;
+    if (
+      this.state.withAccordion &&
+      !this.state.expanded &&
+      rowClassMapper &&
+      !rowClassMapper(value)['row-console']
+    ) {
+      return `${this.overflowCellMaxHeight}px`;
+    }
+    return `${this.rowRef && this.rowRef.current && this.rowRef.current.clientHeight}px`;
+  };
+
+  highlightBlockRef = React.createRef();
+
+  rowRef = React.createRef();
+
+  highLightGridRow() {
+    this.highlightBlockRef.current.scrollIntoView({ behavior: 'smooth' });
+    this.highlightBlockRef.current.classList.add(cx('highlight'));
```

Comment:
Maybe it's better to set classes according to the state/props instead of manipulating DOM directly?

Reply:
Fixed.